docs/api/features/singlefeatureviz.rst

```diff
@@ -0,0 +1,54 @@
+.. -*- mode: rst -*-
+
+RadViz Visualizer
+=================
+
```

Comment:
Thank you so much for adding this new page to the documentation! I really appreciate how comprehensive this PR is! The only catch is that we have adopted a new plot directive style for our documentation [described here](https://github.com/DistrictDataLabs/yellowbrick/blob/develop/docs/README.md#adding-new-visualizers-to-the-docs), which auto-generates the images when the documentation is built. Would you mind taking a crack at using the new style here so that we can be sure updated images are built with the documentation each time?